packages/components/pie-assistive-text/src/assistive-text.scss

```diff
@@ -1 +1,29 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-assistiveText {
+    --assistive-text-color: var(--dt-color-content-default);
+
+    display: flex;
+    padding-block-start: var(--dt-spacing-a);
+    align-items: flex-start;
+    color: var(--assistive-text-color);
+
+    &[variant='success'] {
+        --assistive-text-color: var(--dt-color-content-positive);
+    }
+
+    &[variant='error'] {
+        --assistive-text-color: var(--dt-color-content-error);
+    }
+}
+
+
+.c-assistiveText-content {
+    gap: var(--dt-spacing-b);
+    display: flex;
```

Comment:
there's quite a lot of flex going on here, just to double check is it definitely needed?

Reply:
thank you! just double checked and I don't need the ones at the top anymore 😄 removed 👍